eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -54,23 +54,26 @@ parameters:
     type: object
     default:
       Public:
-        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
         ServiceConnection: azure-sdk-tests
         SubscriptionConfigurationFilePaths:
           - eng/common/TestResources/sub-config/AzurePublicMsft.json
       Preview:
-        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
         ServiceConnection: azure-sdk-tests-preview
+        SubscriptionConfigurationFilePaths:
```

Comment:
UseFederatedAuth default is not set to true in .NET yet so I would wait until that goes in before updating these defaults.

Reply:
Yes this will go in after the default switch.